flyway/postgre/migrations/sql/V1_3_Add_lesson_2_table.sql

```diff
@@ -0,0 +1,76 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TYPE organization_status_type AS ENUM ('active', 'suspend', 'deleted');
+CREATE TYPE user_status_type AS ENUM ('active', 'deactivated', 'deleted');
+CREATE TYPE gender_type AS ENUM ('male', 'female');
+
+CREATE TABLE IF NOT EXISTS sso_organization (
+    organization_id SERIAL PRIMARY KEY NOT NULL,
+    organization_name VARCHAR(255) NOT NULL,
+    status organization_status_type,
+    address VARCHAR(255),
+    contact_number VARCHAR(255),
+    email VARCHAR(255),
+    country VARCHAR(255),
+    state VARCHAR(255),
+    city VARCHAR(255),
+    postal_code VARCHAR(255),
+    website_url VARCHAR(255),
+    kvp JSONB,
+    date_created TIMESTAMPTZ DEFAULT now(),
+    date_modified TIMESTAMPTZ DEFAULT now(),
+);
+
+CREATE TABLE IF NOT EXISTS sso_role (
+    role_id UUID DEFAULT uuid_generate_v4() PRIMARY KEY NOT NULL,
+    role_name VARCHAR(255) NOT NULL,
+    role_description VARCHAR(255),
+    organization_id INT NOT NULL,
```

Comment:
Ở bảng sso_organization thì id đang có kiểu dữ liệu là `SERIAL`?
```suggestion
    organization_id SERIAL NOT NULL,
```